bindings/py/packaging/setup.py

```diff
@@ -186,19 +186,17 @@ def generateExtensions():
   from sys import version_info
   if version_info > (3, 0):
```

Comment:
>= ?

Reply:
> &gt;= ?

means "If the Python version is greater than 3.0"  Actually it should probably be (3, 2) because some of the real early releases of Python 3.x did not have some things we need ... don't remember what they were.  It is unlikely anyone has those early versions.